netadapter/server/grpcserver/protowire/common.go

```diff
@@ -26,6 +27,9 @@ func protoHashesToWire(protoHashes []*Hash) ([]*daghash.Hash, error) {
 }
 
 func wireHashToProto(hash *daghash.Hash) *Hash {
+	if hash == nil {
```

Comment:
I prefer if it'll be outside of wireHashToProto so you won't get it for required fields